core/src/main/scala/org/apache/spark/api/python/SerDeUtil.scala

```diff
@@ -57,11 +57,11 @@ private[spark] object SerDeUtil extends Logging {
     //  };
     // TODO: support Py_UNICODE with 2 bytes
```

Comment:
Yea.. actually, this is the reason why I don't like to leave a todo. Yes, it looks apparently fixed.